app/herdbook.py

```diff
@@ -513,14 +523,22 @@ def issue_certificate(i_number):
     if certificate_exists:
         return jsonify({"response": "Certificate already exists"}), 400
 
-    ind.update(**request.form)
+    ## TODO: Make sure this is unique
+    cert_number = str(random.randint(50001, 99999))
```

Comment:
There is a todo but these would normally be a series, so we should keep track and issue the next.

Reply:
This will be fixed in a future PR.